packages/zowe-explorer/package.nls.json

```
@@ -55,6 +55,7 @@
   "uss.removeSession": "Hide Profile",
   "uss.binary": "Toggle Binary",
   "uss.uploadDialog": "Upload Files...",
+  "uss.uploadDialogBinary": "Upload Files... (Binary)",
```

I think the ellipsis (`...`) is usually appended at the very end of menu items. Should we change it to:

```suggestion
  "uss.uploadDialogBinary": "Upload Files (Binary)...",
```